### ISSUE TITLE
Curse of Doom spawning doomguard at the wrong location.

### ISSUE THREAD

**Issue by theryewall** [LightsHope/issues#934]
The curse of doom spell is not spawning the doomguard at the correct location. When it spawns a doomguard it is supposed to be created at the location of the creature that was killed, however currently the doomguard is summoned at the location of the warlock that casted the curse of doom spell. I found this youtube video of someone using the spell in classic wow, and it shows the doomguard being spawned at the location of the creature. https://www.youtube.com/watch?v=3tYPzOyUPm4

**Comment by Kittnz** [comment]
TBC footage.

**Comment by theryewall** [comment]
Are you saying the video I posted is actually TBC? I can probably find another video from classic if that is your gripe, I'm pretty confident that is the way the spell works but I am not positive.

**Comment by Kittnz** [comment]
Yeah i know, https://youtu.be/3tYPzOyUPm4?t=22 you can see the unit getting as pet.
